src/steps.conf.js

```diff
@@ -18,10 +18,14 @@ module.exports = function ({
     idGenerator = defaultIdGenerator, // <() => <string|number>> - function, that will return new generated id.
     objectsProcessor = {} // objectsProcessor childs could be previded here - see objects.processor.js for more details.
 }) {
-    const id = {
-        value: defaultIdValue || idGenerator.call(this),
-        getId: () => this.value,
-        regenerate: () => {
+    const Id = class {
+        constructor () {
+            this.value = defaultIdValue || idGenerator.call(this);
+        }
+        get id () {
```

Comment:
`stepsConfig.id.id` - sounds a bit strange - maybe it will be better to name it `stepsConfig.id.idValue`?